gdeployfeatures/lv/lv.py

```diff
@@ -121,16 +143,20 @@ def get_mount_data(section_dict, devices, vgnames):
             if not su:
                 # No stripe size given assuming 256
                 su = 256
-            section_dict['opts'] = "-f -K -i size=512 -d sw=%s,su=%sk\
- -n size=8192"%(sw[0],su[0])
+            section_dict['opts'] = "-f -K -i size=512 -d sw=%s,su=%sk"\
+                                   "-n size=8192"%(sw[0],su[0])
```

Comment:
This will fail. Since there is no space between su=%sk and -n ... 
Have you tested this out? Like creating filesystem with this change?